tools/homebrew-start/lanraragi

```diff
@@ -1,22 +1,35 @@
 #!/bin/bash
 
-trap "kill 0" SIGINT
-
-# prefix/place where lrr is installed & modify redis config
+# prefix/place where lrr is installed
 workdir=$(dirname $0)/$(dirname $(readlink $0))/..
 
+# initialize LRR_DATA_DIRECTORY if not set
+if [[ ${LRR_DATA_DIRECTORY} == "" ]]; then
+  if [[ ${OSTYPE} == "darwin"* ]]; then
+    _LRR_DATA_DIRECTORY="${HOME}/Library/Application Support/LANraragi"
+  else
+    _LRR_DATA_DIRECTORY="${HOME}/LANraragi"
+  fi
+  export LRR_DATA_DIRECTORY="${_LRR_DATA_DIRECTORY}"
+fi
+echo "Data Directory is set to '${LRR_DATA_DIRECTORY}'"
+
+# change values in redis.conf
 if [[ ${OSTYPE} == "darwin"* ]]; then
-  _LRR_DATA_DIRECTORY="${HOME}/Library/Application Support/LANraragi"
-  sed -i "" "s~<LRR_DATA_DIRECTORY>~${_LRR_DATA_DIRECTORY}~g" ${workdir}/redis.conf
+  sed -i "" "s~<LRR_DATA_DIRECTORY>~${LRR_DATA_DIRECTORY}~g" ${workdir}/redis.conf
 else
-  _LRR_DATA_DIRECTORY="${HOME}/LANraragi"
-  sed -i"" "s~<LRR_DATA_DIRECTORY>~${_LRR_DATA_DIRECTORY}~g" ${workdir}/redis.conf
+  sed -i"" "s~<LRR_DATA_DIRECTORY>~${LRR_DATA_DIRECTORY}~g" ${workdir}/redis.conf
 fi
-echo "Data Directory is set to '${_LRR_DATA_DIRECTORY}'"
 
-# create path if it does not exist yet
-mkdir -p "${_LRR_DATA_DIRECTORY}/log"
+# initialize LRR_LOG_DIRECTORY if not set
+if [[ ${LRR_LOG_DIRECTORY} == "" ]]; then
+  export LRR_LOG_DIRECTORY="${LRR_DATA_DIRECTORY}/log"
```

Comment:
This is a good opportunity to redirect logs to ~/Library/Logs for darwin:  
`_LRR_LOG_DIRECTORY="${HOME}/Logs/LANraragi"`

Reply:
The correct path is `"${HOME}/Library/Logs/LANraragi"`